esmvaltool/recipes/hydrology/recipe_hype.yml

```diff
@@ -43,8 +43,8 @@ diagnostics:
     variables:
       tas: &var
         mip: day
-        start_year: 1979
-        end_year: 1979
+        start_year: 1990
```

Comment:
This is not related to wflow nor lazy regridding, right?

Reply:
right, similar to wflow recipe, the hype recipe should use realistic time length. Actually, we don't have the data for the year 1979 on Cartesius. I understand that this change is not related to this PR, but just because it was a very small change. Please let me know if it makes a problem, then I will create another PR for this.

Reply:
Okay! I typically think of the hydrology recipes as minimal examples. The more data you require the more testing resources we'll need. And note that our project on Cartesius is not in any way related to ESMValTool, so it'd make more sense to take into account the available data for the testbot at DKRZ. But realistic ranges is fine with me

Reply:
> Okay! I typically think of the hydrology recipes as minimal examples.

We discussed this with @SarahAlidoost and @nielsdrost and we agreed that they should be minimal but working examples, i.e. not crash because you run out of memory as soon as you try to use a realistic amount of input data.

Reply:
I moved hype from this PR to another PR. Also, I made issue #2034 to organize the changes better.